proxy/pkg/proxy/proxy.go

```diff
@@ -379,6 +383,10 @@ func defaultPolicies() []config.Policy {
 					Endpoint: "/signin/",
 					Backend:  "http://localhost:9130",
 				},
+				{
+					Endpoint: "/archiver",
```

Comment:
@jvillafanez I made the proxy route also match `/archiver` so that we can use `curl -u einstein:relativity -k 'https://localhost:9200/archiver?dir=/home/Foo' --output Foo.tar`

Reply:
That's ok for me, although I don't know if we have plans to add additional actions against the archiver service at some point.

For example, if we want to archive huge folder, we might want to provide async actions such as `/archiver/async?dir=folder` to create the request, `/archiver/async/{id}/state` to check the state (queued, archiving fileA, ready...), `/archiver/async/{id}` to download the archive if finished.
This allows more flexibility since we wouldn't need to touch the routing.